packages/react-router-dom/dom.ts

```diff
@@ -243,6 +253,18 @@ export function getFormSubmissionInfo(
       `Cannot submit element that is not <form>, <button>, or ` +
         `<input type="submit|image">`
     );
+  } else if (
+    options.encType !== undefined &&
+    options.encType !== "application/x-www-form-urlencoded"
+  ) {
+    // Any encType other than these (null, application/json, text/plain) means
```

Comment:
FWIW this comment confused me at first because I thought the word "these" referred to the items following it in parentheses, when it's referring to `undefined` and `"application/x-www-form-urlencoded"` used above, right?